source/includes/_downloading.md

```diff
@@ -73,20 +73,24 @@ the data is received. There is a 30s default timeout applied to downloading a
 skylink. If the data can not be found within this 30s time constraint, a `404`
 error will be returned. This timeout is configurable.
 
-### Parameters
+### Parameters (non-BrowserJS)
```

Comment:
should it be called NodeJS ?

Reply:
I don't think so as python and go are also included.